src/middleware/thunk.js

```diff
@@ -1,10 +1,6 @@
-export default function thunkMiddleware(getState) {
-  return (next) => {
-    const recurse = (action) =>
-      typeof action === 'function' ?
-        action(recurse, getState) :
-        next(action);
-
-    return recurse;
-  };
+export default function thunkMiddleware(getState, dispatch) {
```

Comment:
Oh, so if the thunk ain't the first one, `dispatch` will re-start the chain right?